src/debugpy/common/log.py

```diff
@@ -177,8 +198,8 @@ def error(*args, **kwargs):
     return AssertionError(write_format("error", *args, **kwargs))
 
 
-def _exception(format_string="", *args, **kwargs):
-    level = kwargs.pop("level", "error")
+def _exception(format_string="", *args: Any, **kwargs: Any):
```

Comment:
Is there any way to designate `format_string` here and elsewhere as "`str` but must be a literal"? It should never be called with non-literal values as first arguments due to possibility of format string injection attack, and it would be nice to enforce this on type levels.